CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+#					CONTRIBUTING
+
+Contributing to the official buyTogether Application
+Welcome! We're so excited that you're considering contributing to the official GitHub page for buyTogether.
+
+---
+
+## Make changes locally
+
+- Fork the repo so that you can make your changes without affecting the original project until you're ready to merge them.
+
+- Create a working branch and start with your changes!
+
+---
+
+## Commit your update
+Commit the changes once you are happy with them. 
+
+---
+
+## Pull Request
+When you're finished with the changes, create a pull request, also known as a PR.
+
+- Fill the "Ready for review" template so that we can review your PR. This helps reviewers understand your changes as well as the purpose of your pull request.
+- As you update your PR and apply changes, mark each conversation as resolved.
```

Comment:
Unless the comment in the review is very trivial, It's up for the reviewer to resolve the conversation to indicate that they are ok with the updated content.